climada/entity/exposures/crop_production.py

```diff
@@ -366,7 +434,20 @@ def set_mean_of_several_models(self, input_dir=INPUT_DIR, hist_mean=HIST_MEAN_PA
 
         return self
 
-    def set_to_usd(self, input_dir=INPUT_DIR, yearrange=YEARS_FAO):
+    def set_to_kcal(self):
+        """Converts the exposure from tonnes to kcal using conversion factor per crop type.
+
+        Returns:
+            Exposure
+        """
+        if not 't' in self.value_unit:
+            LOGGER.warning('self.unit is neither t nor t / year.')
+        self['tonnes_per_year'] = self['value'].values
```

Comment:
How can the unit be 't' but the value is `tonnes_per_year` ?

Reply:
this will be adjusted